### PR TITLE
fix(runtime): small changes non-functional improvements from runtime

### PR DESCRIPTION
**Pull request by mtulio**
Minimal fixes and improvements found after PRs #10 #11 and #12 :
- Add header to auto-generated Containerfiles* created on build time
- Add VERSION file to image
- Add shared volume to avoid mixing internal files (kubeconfig, aggregator status file, openshift-tests utility, openshift-tests stat file, etc) to non-related with sonobuoy results, but being used in the `SONOBUOY_RESULT_DIR`
- Review function comments
- Review general comments
- Reuse report-pogress function

### CODE REVIEW

**Comment by mtulio** [comment]
Execution has been finished:
```
./openshift-provider-cert-cli results results/aws-integrated-20220615/202206151353_sonobuoy_c12cf73f-035c-4adf-80f4-154734273b7c.tar.gz 
Plugin: openshift-conformance-validated
Status: failed
Total: 3386
Passed: 1412
Failed: 49
Skipped: 1925

Plugin: openshift-kube-conformance
Status: failed
Total: 477
Passed: 474
Failed: 3
Skipped: 0

```

File: openshift-tests-provider-cert/hack/Containerfile.alp [hunk 1]
```diff
@@ -1,3 +1,7 @@
+#
+# ATTENTION: The Containerfile was created automatically from hack/Containerfile.alp
+#
+
```
Comment (bostrt) [inline openshift-tests-provider-cert/hack/Containerfile.alp:4]: Is this comment here to suggest no one edit the file directly?

Comment (mtulio) [reply openshift-tests-provider-cert/hack/Containerfile.alp:4]: @bostrt no one edit the **generated file directly**. Is it better to keep more clear message?
I could change that phrase to an env var on the source file `hack/Containerfile.alp` to be more helpful.

Comment (bostrt) [reply openshift-tests-provider-cert/hack/Containerfile.alp:4]: Here's a comment from the generated code in the cli tool. I think simply saying "do not edit" will suffice:

https://github.com/redhat-openshift-ecosystem/provider-certification-tool/blob/mvp/pkg/assets/bindata.go#L1

Comment (mtulio) [reply openshift-tests-provider-cert/hack/Containerfile.alp]: Thanks @bostrt . changed

**Pull request merged**